pkg/webservice/dao/application_info.go

```diff
@@ -56,3 +56,7 @@ type PlaceholderDAOInfo struct {
 	Replaced      int64            `json:"replaced,omitempty"`
 	TimedOut      int64            `json:"timedout,omitempty"`
 }
+
+type ApplicationIDsDAOInfo struct {
+	AppIDs []string
```

Comment:
Does it need json tag?

Reply:
yes, i will add json tag & keep consistency